requirements.txt

```diff
@@ -7,5 +7,4 @@ PyYAML==5.4.*
 recommonmark==0.7.*
 sphinx_rtd_theme==0.5.*
 opt_einsum==3.3.*
-primme==3.2.*
```

Comment:
It is better to keep the `primme` package here but with a comment  # in order to remind the user to install it if possible.

Reply:
I don't think a comment here will help "remind" any user. It is better added to the installation doc, and the old one is very outdated.

Reply:
at least remind us :joy: